src/eval/operation.rs

```diff
@@ -387,7 +387,7 @@ impl<R: ImportResolver, C: Cache> VirtualMachine<R, C> {
             UnaryOp::Pol() => {
                 if let Term::Lbl(l) = &*t {
                     Ok(Closure::atomic_closure(RichTerm::new(
-                        Term::Bool(l.polarity),
+                        l.polarity.into(),
```

Comment:
That can be addressed into another PR, but I wonder if now that wouldn't make more sense to have polarity be `` [| `Positive, `Negative |] `` instead of a `Bool` in usercode.

Reply:
That's an excellent idea. It really shouldn't be a user-facing change, the label polarity is well hidden behind non-exposed primops.

Reply:
This turns out to be pretty easy to do and it doesn't even require changes to the Nickel standard library. It only ever compares polarities for equality and doesn't observe that they were `Bool`s.